### DISCUSSION TITLE
Torque Integration and Spook

### DISCUSSION THREAD

**Discussion by Anto79-ops** [frenck/spook#39] [Q&A] [answered]
Hi!

Just something I wanted to raise here.

I'm using the [Torque](https://www.home-assistant.io/integrations/torque/) core integration. Its kind of a strange integration, in that the entities are[ not available](https://github.com/home-assistant/core/issues/31166) soon after restart, but when I start the Torque application on my phone and starts logging data, they come back with all their history.

Problem is, of course, Spook tells me that I have automations for entities that don't exist (the Torque entities). So, I'm not sure if you can introduce "Ignore" issue that gets remembered at every restart or perhaps the Torque integration could be updated so that the entities survive a restart. 

Any thoughts on this?

**Comment by frenck** [comment]
The issue is that the Torque seems quite old-school, the issue is that is should be modernized to register its entities, and devices and add unique IDs to them so they end up in the registry.

I have no direct plans to add support for ignoring entities.

**Comment by Anto79-ops** [comment]
Agreed! I can post on that old-issue at Torque and reference this.

Cheers

**Comment by Anto79-ops** [comment]
I think maybe a workaround could be to create (template) sensors that have the unavailable state based on the torque entities in question.

but then spook might say I have (template) sensor that is based on a non-existing entity?

**Comment by Anto79-ops** [comment, marked as answer]
@frenck Ok, the problem was solved by making a template sensor with availability and use that for the automation. Just restarted and Spook did not give that same warning.

So this is good news....until you add the feature where Spook will crawl templates and it will then throws some warnings again ...lol..I hope not. 

thank you sir.
